packages/opentelemetry-cloud-trace-propagator/src/ext-types.d.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+declare module 'hex2dec' {
+  export function hexToDec(hex: string): string;
+  export function decToHex(dec: string): string;
```

Comment:
```suggestion
  export function decToHex(dec: string, options?: {prefix: boolean}): string;
```

See https://github.com/donmccurdy/hex2dec#usage

Note that technically both these functions would return `null` if the input is invalid.

Reply:
done